src/composables/accounts.ts

```diff
@@ -115,47 +169,74 @@ export function useAccounts({ store }: IDefaultComposableOptions) {
     }
   }
 
+  function setActiveAccountByProtocolAndIdx(protocol: Protocol, idx: number) {
+    const accountFound = accountsGroupedByProtocol.value[protocol]
+      ?.find((account) => account.idx === idx);
+    if (accountFound) {
+      setActiveAccountByGlobalIdx(accountFound.globalIdx);
+    }
+  }
+
+  function setMnemonic(val: string) {
```

Comment:
Maybe you could consider more meaningful name than `val`?

Reply:
`mnemonic` is already used. Any suggestions?

Reply:
I'm thinking of `phrase` or `mnemonicPhrase` to indicate what kind of value will be passed here

Reply:
I'll go with `newMnemonic` as agreed on private.